### PR TITLE
fix scheduleReconciler log info

### PR DESCRIPTION
**Pull request by yanggangtony**
Thank you for contributing to Velero!

# Please add a summary of your change
The log info is wrong , we should move it to the above code .
/kind cleanup

# Does your change fix a particular issue?

Fixes #(issue)

# Please indicate you've done the following:

- [x] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [ ] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

File: pkg/controller/schedule_controller.go [hunk 1]
```diff
@@ -199,11 +199,10 @@ func (c *scheduleReconciler) checkIfBackupInNewOrProgress(schedule *velerov1.Sch
 
 	for _, backup := range backupList.Items {
 		if backup.Status.Phase == velerov1.BackupPhaseNew || backup.Status.Phase == velerov1.BackupPhaseInProgress {
+			log.Debugf("Schedule %s/%s still has backups are in InProgress or New state, skip submitting backup to avoid overlap.", schedule.Namespace, schedule.Name)
```
Comment (mateusoliveira43) [inline pkg/controller/schedule_controller.go:202]: NIT should it be "... still has backups **that** are in InProgress..." or "... still has backups in InProgress..."?

Comment (yanggangtony) [reply pkg/controller/schedule_controller.go:202]: Thanks for the quick review.
I am not a english speaker.
Your suggestion is make sense to me.

Comment (yanggangtony) [reply pkg/controller/schedule_controller.go]: Rebased.

File: pkg/controller/schedule_controller.go [hunk 2]
```diff
@@ -199,11 +199,10 @@ func (c *scheduleReconciler) checkIfBackupInNewOrProgress(schedule *velerov1.Sch
 
 	for _, backup := range backupList.Items {
 		if backup.Status.Phase == velerov1.BackupPhaseNew || backup.Status.Phase == velerov1.BackupPhaseInProgress {
+			log.Debugf("%s/%s still has backups that are in InProgress...", schedule.Namespace, schedule.Name)
```
Comment (ywk253100) [inline pkg/controller/schedule_controller.go:202]: Change it to `in InProgress or New`? That's confuse if only contains the `InProgress` state

Comment (yanggangtony) [reply pkg/controller/schedule_controller.go:202]: Got it.

Comment (yanggangtony) [reply pkg/controller/schedule_controller.go]: Rebased, Thanks review.

**Review by ywk253100 — APPROVED** [review]

**Review by shubham-pampattiwar — APPROVED** [review]

**Comment by yanggangtony** [comment]
Thanks

**Pull request merged**